doc/ref_model/chapters/chapter04.md

```diff
@@ -410,6 +410,9 @@ This Cloud Infrastructure Profile is intended to be used for both IT workloads a
 #### 4.2.4.2 Network Intensive Profile
 This Cloud Infrastructure Profile is intended to be used for those applications that has high network throughput requirements (up to 50Gbps).
 
+#### 4.2.4.3 Edge Profile
+This Cloud Infrastructure Profile is intended to be used for Edge use cases that have special requirmenet , as FPGA for vRAN use case which required offloading some RAN functionality.
```

Comment:
I think FPGA is not a requirement, but a solution to a requirement. The requirement here is low latency computing, low latency networking and high throughput.

Reply:
@CsatariGergely Please review my suggestion and see if it addresses your concern.